tests/shell/BashateBearTest.py

```diff
@@ -0,0 +1,28 @@
+from bears.shell.BashateBear import BashateBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+E002_bad = """
+  echo "E002: Has a tab"
+"""
+
+E003_bad = """
+   indent is not multiple of 4
+"""
+long_line = 'a = "{0}"'.format('a' * 100)
+
+BashateBearTest = verify_local_bear(
+    BashateBear,
+    valid_files=(),
+    invalid_files=(long_line,
+                   E002_bad,
+                   E003_bad,),
+    tempfile_kwargs={'suffix': '.sh'})
+
+BashateBearTestIgnoreSetting = verify_local_bear(
+    BashateBear,
+    valid_files=(E002_bad,
+                 E003_bad,
+                 long_line),
+    invalid_files=(),
+    tempfile_kwargs={'suffix': '.sh'},
+    settings={'bashate_ignore': 'E002, E003, E006'})
```

Comment:
Maybe don't ignore one of the errors, to be sure that you bear doesn't accidentally deactivate all warnings if at least one ignore is present ;)
You could actually add another test for that :+1: